### PR TITLE
[PLATFORM-1213] Adjust dialog button min widths, add close buttons to error modals

### PR DESCRIPTION
**Pull request by juhah**
Addresses community Product deploy process improvements in https://streamr.atlassian.net/browse/PLATFORM-1213

> In between the 4th slide of the process and the Deploying spinner modal, the entire layout changes. Size of modal, title alignment, type line height, everything changes, causing an uncomfortable jump in the UI. 

This should be better now with the guided modal:
ezgif-6-e21ef00cf0fd

If guide is skipped, there will be a jump because the modals are different sizes:
ezgif-6-189111d9eed7

I added an "OK" button to error modals, it was in the designs, for example:
Screen Shot 2020-01-29 at 15 58 58

However not to ones where there are external buttons, @mattatgit Should they be added?
Screen Shot 2020-01-29 at 15 59 37

(Note that there is an issue open for already for incorrect styling for the external buttons)

### CODE REVIEW

**Comment by juhah** [comment]
@mattatgit To be clear, I added the error modal buttons only because the issue description expressly says "On the error modal, button should be outline button style not solid color" with a link to the design with the button in it: https://share.goabstract.com/36d2338a-eb7d-4160-8f23-710d27786ceb?mode=design&sha=6737e3c26e9fa4ed2007acc54b160d6e9b6cba51

But with or without the buttons, the modal is dismissable by clicking outside the modal. I can remove the buttons again if that's better.

> If the guide is skipped, we should resize the deploy modal so it matches the size of the previous one. Weird UI jumps should always be avoided.

Is it better to make the shortcut modal bigger or make the deploy modal smaller in this case:
Screen Shot 2020-01-30 at 8 49 52 copy

> Re the Ether needed modal, we should never have lines breaking for single words like that.

Ok, will fix.

File: app/src/marketplace/components/Modal/DeployingCommunityDialog/deployingCommunityDialog.pcss [hunk 1]
```diff
@@ -3,7 +3,8 @@
 .DeployingCommunityDialog {}
 
 .content {
-  padding: 3rem 1rem 1.5rem;
+  padding: 3rem 1rem 2rem;
+  padding-bottom: calc(2rem + 3px);
```
[the remark this answers is not in the document]
Comment (juhah) [reply app/src/marketplace/components/Modal/DeployingCommunityDialog/deployingCommunityDialog.pcss]: Magic number used to make the guided modal and deploy modal the same height... Giving a fixed height to the modal might be better though.

[the remark this answers is not in the document]
Comment (mondoreale) [reply app/src/marketplace/components/Modal/DeployingCommunityDialog/deployingCommunityDialog.pcss]: I guess, yeah. If our `1rem` is still different on mobile/tablet then 3px may only work on desktop. Anyway, we could have a ticket about normalizing modal box sizes and address separately.

**Comment by mattatgit** [comment]
@juhah Re the error modals,  probably a gap between Saori asking for a change and my general priniciples re modals, which perhaps she doesn't always follow. Sorry if two sets of requests are colliding there. Generally we should try to avoid modals with vague / semi-meaningful agreement actions like "Okay". If we have buttons they should where possible have clear verb actions.

Re the modal sizes. we can probably design a spinner modal that matches the Deploy one better in size. Quick version of it here (standalone sketch file) https://www.dropbox.com/s/e7zakqhczuo97i5/reworked_modal.sketch.zip?dl=0

**Comment by mattatgit** [comment]
Screenshot 2020-01-30 22 10 50

Screenshot of standalone sketch file posted above

**Comment by juhah** [comment]
@mattatgit Removed the error dialog buttons and added a minimal version of the deploy spinner:

Guided deploy:
ezgif-2-fb13b0ecae22

Guide skipped:
ezgif-2-c45f58f3c705

**Comment by mondoreale** [comment]
Any chance we can move forward with this, merge possibly? This seems like a button-related PR and those have been addressed, afaik. Can we address modal sizes and the actual button presence/copy in a different ticket and PR? Don't want this one to scope-creep on us.

@mattatgit what if "Ok" was "Close" instead? Sounds more actionable than "Ok", haha. Just a thought.

lmk guys!

**Comment by mattatgit** [comment]
@mondoreale 

> what if "Ok" was "Close" instead? Sounds more actionable than "Ok", haha. Just a thought.

Yep, agreed, "Close" is acceptable, "OK" is not.

**Comment by mondoreale** [comment]
🚀

**Pull request merged**